### DISCUSSION TITLE
Can we write files?

### DISCUSSION THREAD

**Discussion by gotnoshoeson** [Asaye/autocad-dxf#2] [Q&A]
I want to read a file, add new entities to the Entities.entities array and then save the file. I don't see any examples of converting the Entities back into the native DXF format. Is this possible? Is it planned already? Or did I miss something in the docs? Thanks.

**Comment by Asaye** [comment]
Hello,
Thanks for your question. The feature which you wanted is planned to be included in future versions. Unfortunately, the current or previous versions do not have the functionality to convert Entities into native DXF format. 
Thanks,

Yahoo Mail: Search, Organize, Conquer 
 
  On Fri, Dec 6, 2024 at 9:05 PM, Miles ***@***.***> wrote:   


I want to read a file, add new entities to the Entities.entities array and then save the file. I don't see any examples of converting the Entities back into the native DXF format. Is this possible? Is it planned already? Or did I miss something in the docs? Thanks.

**Reply by gotnoshoeson** [reply]
Thanks for the quick response :)
